pyspectral/utils.py

```diff
@@ -222,10 +70,10 @@
                'MTG-I1': 'fci'
                }
 
-HTTP_PYSPECTRAL_RSR = "https://zenodo.org/record/4305549/files/pyspectral_rsr_data.tgz"
+HTTP_PYSPECTRAL_RSR = "https://zenodo.org/record/5797957/files/pyspectral_rsr_data.tgz"
```

Comment:
`HTTPS_PYSPECTRAL_RSR` perhaps?

Reply:
Ah, yes, well. I will leave this for now